### PR TITLE
Add basic UI & Work with props & Routing

### PR DESCRIPTION
**Pull request by MaksimkaSota**
1. Deleted unnecessary in manifest.json;
2. Added basic Components;
3. Added styles using CSS-modules;
4. Added SCSS, SCSS-loader, classnames for comfortable work with styles;
5. Added work with props;
6. Added routing (react-router-dom).

### CODE REVIEW

File: src/Components/App.jsx [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+import { Main } from './Main/Main';
+import { BrowserRouter } from 'react-router-dom';
 
 export const App = () => {
   return (
-    <div className="App">
-      Social Network
-    </div>
+    <BrowserRouter>
+      <div className="wrapper">
```
Comment (ZhekaVasil) [inline src/Components/App.jsx]: I strongly recommend to use sass modules, it will incapsulate your styles and they will affect only desire component =)

**Comment by ZhekaVasil** [comment]
It is 2023, you could use `.js` instead of `.jsx` =)

File: src/Components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <div className={classes.mailContainer}>
+        <img className={classes.mail} src={email} alt="mail" />
+      </div>
+      <div className={classes.contactsContainer}>
+        <h2 className={classes.title}>Contacts:</h2>
+        <div className={classes.contacts}>
+          <a className={classes.contact}
+             href="https://github.com/MaksimkaSota"
+             target="_blank"
+             rel="noreferrer">
```
Comment (ZhekaVasil) [inline src/Components/Footer/Footer.jsx]: It is also good idea to add "noopener" attribute for external links =)
```
rel="noopener noreferrer"
```

File: src/Components/Main/MessagesPage/Dialog/Dialog.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { NavLink } from 'react-router-dom';
+import classes from './Dialog.module.scss';
+import classNames from 'classnames';
```
Comment (ZhekaVasil) [inline src/Components/Main/MessagesPage/Dialog/Dialog.jsx]: you could simplify the name, it can be just "cn" =)

**Review by ZhekaVasil — APPROVED** [review]

File: src/Components/App.jsx (hunk 1 above)
Comment (MaksimkaSota) [reply src/Components/App.jsx]: Done!

File: src/Components/Footer/Footer.jsx (hunk 2 above)
Comment (MaksimkaSota) [reply src/Components/Footer/Footer.jsx]: Done!

File: src/Components/Main/MessagesPage/Dialog/Dialog.jsx (hunk 3 above)
Comment (MaksimkaSota) [reply src/Components/Main/MessagesPage/Dialog/Dialog.jsx]: Done!

**Pull request merged**